services/app-api/src/emailer/emails/sendQuestionCMSEmail.ts

```diff
@@ -0,0 +1,78 @@
+import { packageName as generatePackageName } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+import { formatCalendarDate } from '../../../../app-web/src/common-code/dateHelpers'
+import { pruneDuplicateEmails } from '../formatters'
+import type { EmailConfiguration, EmailData, StateAnalystsEmails } from '..'
+import type { ProgramType, CMSUserType } from '../../domain-models'
+import {
+    stripHTMLFromTemplate,
+    renderTemplate,
+    findContractPrograms,
+} from '../templateHelpers'
+import { submissionQuestionResponseURL } from '../generateURLs'
+import type { ContractRevisionWithRatesType } from '../../domain-models/contractAndRates'
+
+export const sendQuestionCMSEmail = async (
+    contractRev: ContractRevisionWithRatesType,
+    stateAnalystsEmails: StateAnalystsEmails,
+    cmsRequestor: CMSUserType,
+    config: EmailConfiguration,
+    statePrograms: ProgramType[],
+    dateAsked: Date,
+    roundNumber: number
```

Comment:
Not sure if we wanted to do this, but if we passed in the `question` data from the resolver, we could reduce the parameters by 2. The `question` data will have `addedBy` and `createdAt` for the CMS requestor email and the date the question was asked. Also, we then won't need to generate `dateAsked` using `new Date()` in the resolver

cc: @macrael

Reply:
I feel like these functions have gone back and forth on what they accept over time. In general I'm down for stuff like Question and Contract being passed directly and the emailer being responsible for pulling out what matters for the email, rather than the resolvers pulling that data out themselves. But I don't care that much either way. It used to be inconsistent between email handlers which is probably not ideal. I do think part of why this went this way was b/c of the complication of parsing protobuf stuff which we now hopefully are avoiding